src/webargs/falconparser.py

```diff
@@ -13,6 +13,9 @@
 status_map = {422: HTTP_422}
 
 
+FALCON_VERSION = falcon.__version__[0]
+
+
```

Comment:
On second thought, I'd write

    FALCON_VERSION_INFO = tuple(LooseVersion(falcon.__version__).version)

so that we get version as integer.

(We could also cast to `int`. I don't really mind.)